### DISCUSSION TITLE
March Melbourne MicroPython Meetup videos

### DISCUSSION THREAD

**Discussion by mattytrentini** [micropython/micropython#11139] [General]
Hello folks, the March Melbourne MicroPython Meetup videos are now available:

Damien on [v1.20 and BLE on the RP2040](https://www.youtube.com/watch?v=fH33D6ER7ls)

Matt giving the usual monthly [News Roundup](https://www.youtube.com/watch?v=DyRCR5ALp38). 
[Slides are also available](https://docs.google.com/presentation/d/e/2PACX-1vRS0SJE0zdLK0jolkyyO2bYPGd5x9dxop8SWe6vMNsNLfb9qxEVeSrYcBYpI_gmf6sxnWkdEtNR-IQ_/pub).

**Comment by robert-hh** [comment]
I just noticed the v1.20 tag. No more "we'll have v1.20 by the end of the month".

**Reply by peterhinch** [reply]
The maintainers' prayer: "Oh Lord give us a two-monthly release cycle. But not just yet."

With apologies to St. Augustine. :-)
